### ISSUE TITLE
Use system proxy and/or support configuring proxy options

### ISSUE THREAD

**Issue by leschekfm** [jiahaog/nativefier#715]
### Description

AFAIK electron or the used chrome instance use by default the network settings provided by the operating system and therefore support configuring a proxy out of the box. 

The package I created through nativefier seems to ignore the OS's configuration. Other electron based apps like visual studio code or slack indeed do pick the proxy settings up though.

Is this a behavior that should be expected or  is it possible to configure a proxy any other way (didn't find any options related to this)?

### Steps to reproduce issue

1. create a nativefied app
2. start the app and expect everything to work normally
3. configure your OS to use a non existent proxy
4. start the app and expect requests to load the page fail

### Details

- Are you nativefying a *public* website?
yes
- Feature request? Have you looked at `nativefier --help` to see if an existing option could fit your needs?
no / yes
- Full nativefier command used to build your app: `nativefier https://slack.com --honest --maximize --single-instance --platform windows --arch x64`
- Version of Nativefier (run `nativefier --version`): `v7.6.8`
- Version of node.js (run `node --version`): `v9.3.0`
- OS: `Windows 10 - x64`

**Comment by ronjouch** [comment]
Yes, that would be nice. Closing as duplicate of https://github.com/jiahaog/nativefier/issues/361 , help welcome.

**Comment by leschekfm** [comment]
@ronjouch I don't think this is a duplicate. The other issue mentions that  
> I am behind a local proxy, so building fails with a connection error.

In contrast my issue is with an already successfully built app not using the default behavior electron provides.

>Yes, that would be nice.

I'm not asking for a feature. To what I know this should be working out of the box. At least the [electron docs](https://electronjs.org/docs/api/chrome-command-line-switches#--proxy-serveraddressport) hint that, if I'm not getting this wrong.
> Use a specified proxy server, which overrides the system setting.

Therefore I'd like to know whether the behavior is overridden in any way (didn't find anything related in the code) and if so why.

**Comment by leschekfm** [comment]
@ronjouch I need to politely disagree as these are completely separate issues.

1. The other issue wants to run nativefier **_while creating an app bundle_** behind a proxy. This is something that needs to be implemented and is therefore a feature request.

2. What I want is to know why the build result **_after running nativefier_** does not use the network connection provided by the OS. I verified that electron does use that by default. You can validate that too by running https://github.com/electron/simple-samples/tree/master/url and then adjusting your network connection like mentioned in the STRs.

If this is no intentional behavior and the base library electron does provide the functionality, I think this should be handled as a bug.

Is the original author @jiahaog still working on the project and can maybe give some insight?

**Comment by ronjouch** [comment]
@leschekfm hey you're right! 👍 for insisting, re-opening.

The original author, Jia, isn't active these days And I am only doing super-minimal maintainance: triaging issue (and sometimes failing at it 😄), bumping dependencies & default Electron once in a while, and merging reasonable PRs. So don't count on this (or any other open bug) being fixed anytime, but contributions welcome.

**Comment by davide-sergi** [comment]
Any updates on this?
